### ISSUE TITLE
Fetching errors

### ISSUE THREAD

**Issue by JuLLT** [freescout-helpdesk/freescout#2522]
Using Freescout since few days, seems to be running well except we encounter a lot of fetching issues.

Below are errors in logs, there are 11 mailboxes - errors happen randomly on 1 or 2 boxes every 2 or 3 minutes :
Error: imap_close(): supplied resource is not a valid imap resource;
Folder: INBOX; Error: imap_reopen(): Couldn't re-open stream;

Sometimes but the most common errors in log is above :
Error: imap_open(): Couldn't open stream {mail.xxx.fr:995/pop3/ssl}. Can not authenticate to POP3 server: POP3 connection broken in response;

Any settings to have it working fine ? Thank you

**Comment by freescout-helpdesk** [comment]
Most likely connection issue between your server and mail server. Try to move the server to another location or try some other hosting.

**Comment by JuLLT** [comment]
We already tried several things and we did not had this kind of problems with other solutions.

Less error using IMAP instead of POP3 and unboxing "validate the certificate". Only this error into the log :
Error: imap_open(): Couldn't open stream {mail.xxx.fr:993/imap/novalidate-cert/ssl}. Can not authenticate to IMAP server: [CLOSED] IMAP connection broken (authenticate);

The problem is that sometimes it lock the process, I need to clear the cache, kill process and use the manual link to run (cron). Is there something to avoid that or the relaunch automatically the process ?
